From 38db1b6da970885b161583f5881bc8956e5810bf Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Fri, 5 Jan 2018 17:40:56 -0800
Subject: [PATCH 03/10] dma: xilinx: dpdma: ratelimit the error interrupt
 message

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit 644cd15d989779630ea3364f9c085469e2ac703a)
---
 drivers/dma/xilinx/xilinx_dpdma.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/dma/xilinx/xilinx_dpdma.c b/drivers/dma/xilinx/xilinx_dpdma.c
index c155dcd..0964f19 100644
--- a/drivers/dma/xilinx/xilinx_dpdma.c
+++ b/drivers/dma/xilinx/xilinx_dpdma.c
@@ -1901,8 +1901,9 @@ static void xilinx_dpdma_handle_err_intr(struct xilinx_dpdma_device *xdev,
 	bool err = xilinx_dpdma_err(isr, eisr);
 	unsigned int i;
 
-	dev_err(xdev->dev, "error intr: isr = 0x%08x, eisr = 0x%08x\n",
-		isr, eisr);
+	dev_dbg_ratelimited(xdev->dev,
+			    "error intr: isr = 0x%08x, eisr = 0x%08x\n",
+			    isr, eisr);
 
 	/* Disable channel error interrupts until errors are handled. */
 	dpdma_write(xdev->reg, XILINX_DPDMA_IDS,
-- 
1.8.5.2.233.g932f7e4

